dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/MavenDependencyMediation.java

```diff
@@ -44,7 +46,7 @@ public AnnotatedClassPath mediate(DependencyGraph dependencyGraph) {
 
       File file = artifact.getFile();
       if (file != null && file.getName().endsWith(".jar")) {
-        String versionlessCoordinates = Artifacts.makeKey(artifact);
+        String versionlessCoordinates = Artifacts.makeKey(artifact) + artifact.getClassifier();
```

Comment:
Does `getClassifier()` return the colon prefix, too? Like ":test"?

Reply:
No, it doesn't give a colon prefix. Let me add one. I think it's more clear.